.github/workflows/build.yml

```diff
@@ -146,8 +146,73 @@ jobs:
     - name: Image digest
       run: echo ${{ steps.build.outputs.digest }}
 
+  build-windows:
+    needs: version
+    runs-on: windows-latest
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        base-image:
+          - mcr.microsoft.com/windows/servercore:ltsc2022
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
```

Comment:
```suggestion
        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Reply:
Yeah this should be pinned above as well, feel free to change it.
Dependabot can update this format and change the comment indicating the version